README.md

```diff
@@ -121,6 +121,7 @@ prebuild [options]
   --upload      -u  [gh-token]  (upload prebuilds to github)
   --upload-all  -u  [gh-token]  (upload all files from ./prebuilds folder to github)
   --preinstall  -i  script      (run this script before prebuilding)
+  --precompress -c  script      (run this script before compression)
```

Comment:
I'm :-1: on introducing the precompress script in the same pull request as cmake-js.

Reply:
I just noticed it was two PRs, my bad.

Reply:
Yeh sorry it's a bit messy as this PR is a branch off the cmake one